README-TEST.md

```diff
@@ -169,3 +169,36 @@ Files in ClusterTest
 * src/main/scala//net/snowflake/spark/snowflake/ClusterTest.scala: The driver class to run test
 * src/main/scala//net/snowflake/spark/snowflake/testsuite/ClusterTestSuiteBase.scala: Test case base class.
```

Comment:
Please add the two new test cases file name here and short description.

Reply:
Nice catch, thanks!